README.md

```diff
@@ -54,7 +53,7 @@ Ensure you are installing Node 10 or greater and npm 6 or greater.
 
 ### Installing Docker
 
-Click [here](https://docs.docker.com/v17.12/install/) for the Docker installation site.  Scroll down to "Supported Platforms" and follow the instructions to download & install Docker Desktop for your operating system (or Docker CE for linux).
+See the [Docker installation](https://docs.docker.com/v17.12/install/) "Supported Platforms" section and follow the instructions to download & install Docker Desktop for your operating system (or Docker CE for linux).
```

Comment:
```suggestion
See the [Docker installation](https://docs.docker.com/install/) "Supported platforms" section and follow the instructions to download & install Docker Desktop for your operating system (or Docker CE for linux).
```

Can be fixed in a separate PR, but the current Docker install link here (for v17.12) is out of date - this link ☝️ will always link to the latest release.

Reply:
Good suggestion.

The original change was to reword it to avoid "Click here", so if this is changed in another PR then my preference is to [avoid "Click here" links](https://www.w3.org/QA/Tips/noClickHere).